testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/mgmt/JMSBridgeManagementTestCase.java

```diff
@@ -0,0 +1,351 @@
+    static final PathAddress ADDRESS_NETTY_ACCEPTOR = ADDRESS_HORNETQ.append(PathElement.pathElement("remote-acceptor", "netty"));
+
+    private List<ServerSetupTask> tasks = new ArrayList<ServerSetupTask>(4);
+    private List<ServerSetupTask> tasksToClean = new ArrayList<ServerSetupTask>(4);
+    
+    public JMSBridgeManagementTestCase(){
+        tasks.add(new DisableHornetQSecruity());
+        tasks.add(new SetupNetty());
+        tasks.add(new SetupConnectionFactory());
+        tasks.add(new SetupQueuesFactory());
+        tasks.add(new SetupJMSBridge());
+    }
+
+    @Before
+    public void setup() throws Exception{
+        for(ServerSetupTask t:this.tasks){
+            this.tasksToClean.add(t);
+            t.setup(this.managementClient, "fake");
+        }
+    }
+
+    @After
+    public void cleanup(){
+        try{
+            for(ServerSetupTask sst:this.tasksToClean){
+                sst.tearDown(this.managementClient, "fake");
+            }
+        }catch(Exception e){
+            
+        }
+        this.tasksToClean.clear();
+    }
+
+    @Test
+    public void testStopStart() throws Exception {
+        //actually even checking state seems enough, but lets try a circle.
+        reload(this.managementClient);
+        final ModelControllerClient client = this.managementClient.getControllerClient();
+        ModelNode operation = Util.createOperation(READ_ATTRIBUTE_OPERATION, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        operation.get(INCLUDE_RUNTIME).set(true);
+        operation.get(NAME).set(STARTED);
+        ModelNode result = client.execute(operation);
+        Assert.assertTrue("Bridge should start!",result.get(RESULT).asBoolean());
+        
+        operation = Util.createOperation(STOP, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        result = client.execute(operation);
+        operation = Util.createOperation(READ_ATTRIBUTE_OPERATION, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        operation.get(INCLUDE_RUNTIME).set(true);
+        operation.get(NAME).set(STARTED);
+        result = client.execute(operation);
+        Assert.assertFalse("Bridge should stop!",result.get(RESULT).asBoolean());
+
+        operation = Util.createOperation(START, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        result = client.execute(operation);
+        operation = Util.createOperation(READ_ATTRIBUTE_OPERATION, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        operation.get(INCLUDE_RUNTIME).set(true);
+        operation.get(NAME).set(STARTED);
+        result = client.execute(operation);
+        Assert.assertTrue("Bridge should start!",result.get(RESULT).asBoolean());
+    }
+
+    @Override
+    protected ModelControllerClient getModelControllerClient() {
+        return this.managementClient.getControllerClient();
+    }
+    public static void reload(final ManagementClient managementClient) throws Exception {
```

Comment:
Maybe because it does not work? Only domain mode test use this "blocking" mode, which is supported in code: host-controler/.../ServerReloadHandler  (while command name is RESTART ??)

However, the server/...../ServerProcessReloadHandler does not allow such parameters to be passed. 

[standalone@localhost:9990 /] :reload(blocking="true")
'blocking' is not found among the supported properties: [admin-only, use-current-server-config]


But just to test, Ive updated test to use given command, still I had to add timeout-wait loop.


One other thing I dont understand, this command is not listed:

[domain@localhost:9990 /] cd host=master/server=server-one         
[domain@localhost:9990 server=server-one] :read-operation-names
{
    "outcome" => "success",
    "result" => [
        "clean-obsolete-content",
        "list-add",
        "list-clear",
        "list-get",
        "list-remove",
        "map-clear",
        "map-get",
        "map-put",
        "map-remove",
        "product-info",
        "read-attribute",
        "read-attribute-group",
        "read-attribute-group-names",
        "read-children-names",
        "read-children-resources",
        "read-children-types",
        "read-config-as-xml",
        "read-operation-description",
        "read-operation-names",
        "read-resource",
        "read-resource-description",
        "resolve-expression",
        "resolve-internet-address",
        "upload-deployment-bytes",
        "upload-deployment-stream",
        "upload-deployment-url",
        "validate-operation"
    ]
}